.genx/templates/web-components/component/component.hbs

```diff
@@ -0,0 +1,17 @@
+import { User } from '@genesislcap/foundation-user';
+import { customElement, GenesisElement } from '@genesislcap/web-core';
+import { {{pascalCase tile.title}}Styles as styles } from './{{kebabCase tile.title}}.styles';
+import { {{pascalCase tile.title}}Template as template } from './{{kebabCase tile.title}}.template';
+
+@customElement({
+  name: '{{kebabCase tile.componentName}}',
+  template,
+  styles,
+})
+export class {{pascalCase tile.componentName}} extends GenesisElement {
+  @User user: User;
+
+  constructor() {
+    super();
```

Comment:
nit: I'd just remove this bit

Reply:
[68a6087](https://github.com/genesiscommunitysuccess/blank-app-seed/pull/275/commits/68a6087374efa86b795123814f3f9e282506a562)